.github/workflows/review-bot.yaml

```diff
@@ -0,0 +1,44 @@
+name: review bot
+on:
+  issue_comment:
+    types: [created, edited]
+jobs:
+  review_comment:
+    if: contains(github.event.comment.body, '/review-bot ')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: "DamianReeves/write-file-action@master"
+        with:
+          path: /tmp/comment.txt
+          write-mode: overwrite
+          contents: ${{ github.event.comment.body }}
+      - uses: jc-lab/shim-review-bot@v0.0.8
```

Comment:
A bit nitpick, but in the context of a security review I'd suggest pinning on to a - arguably less readable - commit id.
This is to ensure nobody would rewrite the tag on the - external - action repository and sneak in an altered/deceptive review.

I guess same issue with the `write-file-action` and swap branch name to a commit id.